backend/business-partner-agent/src/test/resources/files/send-proof/02-prover-request-received.json

```diff
@@ -1,7 +1,7 @@
 {
     "updated_at": "2020-09-07 09:19:53.108674Z",
     "thread_id": "00107a82-d4fd-4d8d-a985-4cbbc6ea624d",
-    "initiator": "self",
```

Comment:
why?

Reply:
if request_received and initiator=self, That is a presentation proposal and the verifier has responded to accept the proposal. I've made changes in `handleProofRequestEvent` that will automatically try and construct the proof and respond immediately, but that's the case the test was trying to test.

Reply:
This is a single file within a series used in a test that tests the state transitions in the request proof flow, all other files have initiator self. This does not make sense, and basically means adding a new test case.